packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/bedrock/invoke-model.ts

```diff
@@ -155,7 +157,21 @@ export class BedrockInvokeModel extends sfn.TaskStateBase {
       }),
     ];
 
-    if (this.props.input !== undefined && this.props.input.s3Location !== undefined) {
+    if (this.props.inputPath !== undefined) {
+      policyStatements.push(
+        new iam.PolicyStatement({
+          actions: ['s3:GetObject'],
+          resources: [
+            Stack.of(this).formatArn({
+              region: '',
+              account: '',
```

Comment:
for my own understanding, why are region and account `''`?

Reply:
afaik, this is for global bucket.